From a43b9711e30d7fa55a3c9fd9f1daf957368cece8 Mon Sep 17 00:00:00 2001
From: Bernd Zeimetz <bernd@bzed.de>
Date: Fri, 29 Nov 2013 01:40:55 +0100
Subject: [PATCH] Use GPS_PATH_MAX instead of PATH_MAX.

ntpshm.c was still using PATH_MAX, resulting in a build failure on
architectures like hurd-i386.
---
 ntpshm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ntpshm.c b/ntpshm.c
index 52367d1..88f420b 100644
--- a/ntpshm.c
+++ b/ntpshm.c
@@ -312,7 +312,7 @@ static void init_hook(struct gps_device_t *session)
 /* for chrony SOCK interface, which allows nSec timekeeping */
 {
     /* open the chrony socket */
-    char chrony_path[PATH_MAX];
+    char chrony_path[GPS_PATH_MAX];
 
     session->chronyfd = -1;
     if ( 0 == getuid() ) {
-- 
1.8.4.rc3

